web/rest/user/features/my/dashboard/getDashboard.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Retrieve dashboard
+
+  @createSchema
+  Scenario: Retrieve user dashboard json
+    Given I add User Authorization header
+     When I add "Accept" header equal to "application/json"
+      And I send a "GET" request to "my/dashboard"
+     Then the response status code should be 200
+      And the response should be in JSON
+      And the header "Content-Type" should be equal to "application/json; charset=utf-8"
+      And the JSON should be equal to:
+      """
+      {
+          "userName": "alice@democompany.com",
```

Comment:
Alice?

Reply:
![image](https://github.com/irontec/ivozprovider/assets/101095813/c9e6b2aa-4eed-48be-8db5-7a2daf627b3f)

Reply:
That's not the user name, that's the email. You login with your email because it's unique but you're suposed to show user name here.